src/compile.js

```diff
@@ -190,6 +197,16 @@ function compile(mediaType, apiElements, filename) {
       annotations = annotations.concat(result.annotations);
     });
 
+  if (mediaType === 'text/vnd.apiblueprint') {
+    // Fixing 'multipart/form-data' bodies coming from the API Blueprint
+    // parser: https://github.com/apiaryio/api-blueprint/issues/401
+    [].concat(...transactions.map(t => [t.request, t.response]))
```

Comment:
Could be simpler using [`flat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) (or maybe even using reduce).

```js
transactions
  .map(t => [t.request, t.response])
  .flat()
  .filter(isMultipart)
  .forEach...
```

Reply:
Yes! Looks much better. I forgot about `flat()`. In my commit I removed this part completely though.